### PR TITLE
Update to v0.14.0-rc3

### PR DESCRIPTION
**Pull request by JordanMartinez**
Backlinking to purescript/purescript#3942

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ node_js: stable
 env:
   - PATH=$HOME/purescript:$PATH
 install:
-  - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
+  # - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
```
Comment (kl0tl) [inline .travis.yml:8]: Is the tag defined elsewhere? Won’t this make the download URL invalid otherwise?

Comment (thomashoneyman) [reply .travis.yml:8]:
```suggestion
  # - TAG=$(wget -q -O - https://github.com/purescript/purescript/releases/latest --server-response --max-redirect 0 2>&1 | sed -n -e 's/.*Location:.*tag\///p')
  - TAG=v0.14.0-rc3
```

Comment (thomashoneyman) [reply .travis.yml:8]: Yea, we need to include a new line with the tag.

Comment (JordanMartinez) [reply .travis.yml:8]: Weird... Guess my script didn't properly account for this situation... I've pushed a commit to fix it.

**Comment by kl0tl** [comment]
The build fails because of an old version of `purescript-psa`, it should be upgraded to v0.8.0.

**Comment by JordanMartinez** [comment]
Fixed. Not sure why my script didn't do this properly...

I also updated pulp as that would have caused failure, too.

**Review by kl0tl — APPROVED** [review]

**Pull request merged**